### PR TITLE
refactor(HACBS-2550): use git resolver for verify-ec task

### PR DESCRIPTION
**Pull request by johnbieren**

### CODE REVIEW

File: catalog/pipeline/deploy-release/0.7/README.md [hunk 1]
```diff
@@ -12,10 +12,15 @@ Tekton pipeline to verify Snapshot prior to Deployment
 | releasestrategy | The namespaced name (namespace/name) of the releaseStrategy | No | - |
 | snapshot | The namespaced name (namespace/name) of the snapshot | No | - |
 | enterpriseContractPolicy | JSON representation of the policy to be applied when validating the enterprise contract | No | - |
-| verify_ec_task_bundle | The location of the bundle containing the verify-enterprise-contract task | No | - |
+| verify_ec_task_git_url | The git repo url of the verify-enterprise-contract task | No | - |
```
Comment (scoheb) [inline catalog/pipeline/deploy-release/0.7/README.md:15]: I know the previous pipeline versions all had the EC params with '-' in them...but they are not inline with ours which use camelCase. Should we use this opportunity to change it?

Comment (johnbieren) [reply catalog/pipeline/deploy-release/0.7/README.md:15]: We pull them straight from here https://github.com/redhat-appstudio/infra-deployments/blob/main/components/enterprise-contract/kustomization.yaml#L13
We could manipulate them, but I don't think we want to

**Review by mmalina — APPROVED** [review]
LGTM. But this requires changes in the release-service to supply the right params when creating the pipelineruns, right? So these two changes should be done in sync somehow. I don't see anything about the release-service changes in the acceptance criteria in Jira though.

(And it's another example of a breaking change that once applied in release-service, all the previous pipeline versions will be broken anyway. Hence why keeping pipeline versions is not very useful anyway. Nothing to do with your change, just saying.)

Also, I would appreciate if you added a brief explanation on why this is done in the PR description / commit message. (I know it's in Jira, but still.)

**Comment by johnbieren** [comment]
> LGTM. But this requires changes in the release-service to supply the right params when creating the pipelineruns, right? So these two changes should be done in sync somehow. I don't see anything about the release-service changes in the acceptance criteria in Jira though.
> 
> (And it's another example of a breaking change that once applied in release-service, all the previous pipeline versions will be broken anyway. Hence why keeping pipeline versions is not very useful anyway. Nothing to do with your change, just saying.)
> 
> Also, I would appreciate if you added a brief explanation on why this is done in the PR description / commit message. (I know it's in Jira, but still.)

My plan was to create the new pipeline versions, then create an e2e-tests PR to update the pipeline versions used and pair it with the release-service pr. Since e2e doesn't use latest pipelines, I figured merging this first would be fine (and is kind of the only way to do it)

**Comment by mmalina** [comment]
> My plan was to create the new pipeline versions, then create an e2e-tests PR to update the pipeline versions used and pair it with the release-service pr. Since e2e doesn't use latest pipelines, I figured merging this first would be fine (and is kind of the only way to do it)

Sounds good. Except you didn't add new pipeline versions in this PR... But I see they were all added in https://github.com/redhat-appstudio/release-service-bundles/pull/183 and nobody updated e2e for that yet I suppose. So probably it's ok in this case.

**Comment by johnbieren** [comment]
> > My plan was to create the new pipeline versions, then create an e2e-tests PR to update the pipeline versions used and pair it with the release-service pr. Since e2e doesn't use latest pipelines, I figured merging this first would be fine (and is kind of the only way to do it)
> 
> Sounds good. Except you didn't add new pipeline versions in this PR... But I see they were all added in #183 and nobody updated e2e for that yet I suppose. So probably it's ok in this case.

Yeah, I talked with Rob and his change doesn't have any effect yet (until he completes some other work) so I said I would combine with him and take care of updating e2e

**Pull request merged**